build_tools/pkgci/build_linux_packages.sh

```diff
@@ -165,6 +165,12 @@ function run_in_docker() {
           build_iree_compiler
           run_audit_wheel "iree_base_compiler${package_suffix}" "${python_version}"
           ;;
+        iree-pjrt-plugin-*)
+          pjrt_platform=${package##*-}
+          clean_wheels "iree_pjrt_plugin_${pjrt_platform}" "py3-none"
```

Comment:
See my other comment about whether this code should live in this script or not, but I would like to say: this bash scripting to pull off the `cpu` / `cuda`, etc. suffix from `iree-pjrt-plugin-cpu`/`iree-pjrt-plugin-cuda`, etc. is quite nice :D